### DISCUSSION TITLE
Introduction and Looking for Beginner Issues

### DISCUSSION THREAD

**Discussion by justtcallmejayy** [InternetHealthReport/gsoc#49] [General]
Hi Everyone! 👋

My name is Jay, and I’m excited to start contributing to InternetHealthReport. I’m new to open-source and looking for some beginner-friendly issues to work on. My background is in web development(HTML, CSS, BOOTSTRAP), Python, JavaScript, Java, C#, SQL and I’m eager to help in any way I can!

I’ve looked through the issues and didn’t find any open ones at the moment. Is there anything specific I could assist with? Perhaps documentation, testing, or any other tasks that would be helpful to the community?

Looking forward to collaborating with all of you! 😊

Thanks,  
**Jay Choksi**

**Comment by dpgiakatos** [comment]
Hello Jay,

Sorry for the delayed response!
There are several projects within the Internet Health Report organization that have open issues you can contribute to.

Thank you!
